src/models/room.ts

```diff
@@ -441,6 +458,19 @@ export class Room extends TypedEventEmitter<EmittedEvents, RoomEventHandlerMap>
         return Promise.allSettled(decryptionPromises) as unknown as Promise<void>;
     }
 
+    /**
+     * Gets the creator of the room
+     * @returns {string} The creator of the room, or null if it could not be determined
+     */
+    public getCreator(): string | null {
+        const createEvent = this.currentState.getStateEvents(EventType.RoomCreate, "");
+        if (!createEvent) {
+            return null;
+        }
+        const roomCreator = createEvent.getContent()['creator'];
+        return roomCreator;
```

Comment:
This technically doesn't match its signature, creator could be `undefined`

```suggestion
        return createEvent?.getContent()['creator'] ?? null;
```